### PR TITLE
Add Zooz ZEN55

### PR DESCRIPTION
**Pull request by adidalal**
Source: https://www.support.getzooz.com/kb/article/1391-zen55-dc-signal-sensor-change-log/
Firmware from: https://www.support.getzooz.com/kb/article/1158-zooz-ota-firmware-files/

If this looks OK, happy to add a couple more.

### CODE REVIEW

File: firmwares/zooz/ZEN55.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+	"devices": [
+		{
+			"brand": "Zooz",
+			"model": "ZEN55",
+			"manufacturerId": "0x027a",
+			"productType": "0x0004",
+			"productId": "0x0110"
```
Comment (TSHAdmin) [inline firmwares/zooz/ZEN55.json:8]: "firmwareVersion": {
				"min": "1.0",
				"max": "1.255"
			}

File: firmwares/zooz/ZEN55.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		}
+	],
+	"upgrades": [
+		{
+			"version": "1.30",
```
Comment (TSHAdmin) [inline firmwares/zooz/ZEN55.json:13]: version: "x.y.z" format
For Zooz product details contact @TSHAdmin

Comment (TSHAdmin) [reply firmwares/zooz/ZEN55.json:13]: "version": "1.30.0"

**Comment by adidalal** [comment]
Sorry for the delay, you can squash and merge now @AlCalzone 
Thanks @TSHAdmin!

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**